data_source_simulator/binance_simulator/test_data/data-point-sample.csv

```diff
@@ -1,4 +1,4 @@
-EUR-USD,GBP-USD,JPY-USD,SEK-USD,NTN-USD,AUD-USD,CAD-USD
+EUR-USD,GBP-USD,JPY-USD,SEK-USD,NTN-USDC,AUD-USD,CAD-USD
```

Comment:
```suggestion
EUR-USD,GBP-USD,JPY-USD,SEK-USD,AUD-USD,CAD-USD
```

As this is the binance plugin, removing `NTN-USDC` as this will jus throw errors atm.

Reply:
> As this is the binance plugin, removing `NTN-USDC` as this will jus throw errors atm.

Actually this is a data source simulator which implements the data spec of Binance, thus we can simulate any symbols in a playbook on it for testing, let's keep it there. It wouldn't impact anything on the plugins side at all.

Reply:
ok